src/gmp.c

```diff
@@ -13112,8 +13112,13 @@ handle_get_info (gmp_parser_t *gmp_parser, GError **error)
       switch (ret)
         {
         case 1:
-          if (send_find_error_to_client ("get_info", "type",
-                                         get_info_data->type,
+          if (send_find_error_to_client ("get_info",
+                                         get_info_data->name
+                                          ? "name"
```

Comment:
I think this would break if name/id contained a ".

It was safe for type because it's checked above.

Reply:
I think this is a general issue with `send_find_error_to_client` that was only avoided so far because of the id validation in GSA, so I'll change that function to XML escape the input.

Reply:
The issue should be addressed with 9504567.